src/test/java/os/failsafe/executor/MultipleNodesShould.java

```diff
@@ -127,12 +126,8 @@ private long countFinishedTasks(TaskExecutionListener l) {
     }
 
     private void awaitEmptyTaskTable() {
-        while (taskRepository.findAll().size() > 0) {
-            try {
-                Thread.sleep(10);
-            } catch (InterruptedException e) {
-                e.printStackTrace();
-            }
-        }
+        await().atMost(Duration.ofSeconds(1))
```

Comment:
You can simply write await().until(...). atMost and pollInterval are not mandatory. I just wanted to make it explicit.